include/mimalloc-types.h

```diff
@@ -136,10 +136,11 @@ typedef union mi_page_flags_u {
   uint16_t value;
   uint8_t  full_aligned;
   struct {
-    bool in_full:1;
-    bool has_aligned:1;
+    unsigned in_full:1;
```

Comment:
Can you use `uint8_t` instead of `unsigned` everywhere. Thanks!

Reply:
Does that really make sense? When you say `uint8_t`, you specify an 8-bit type, but the `:1` ridicules that specificity.

Reply:
Hmm, yes, but in a way, `unsigned` also has some specific bit size. With `uint8_t` we convey more clearly that we want all those bit fields in an 8-bit byte.

Reply:
I tested this, and it seems that it compiles, so I will abide by your wish ;-)

Reply:
Correction. I tested with GCC (which did not have a problem to begin with). Now I tested again with VS2019, and it produces this:

```
[...]\mimalloc\mimalloc-types.h(139): error C2220: the following warning is treated as an error
[...]\mimalloc\mimalloc-types.h(139): warning C4214: nonstandard extension used: bit field types other than int
[...]\mimalloc\mimalloc-types.h(140): warning C4214: nonstandard extension used: bit field types other than int
```

So I am afraid that I won't abide by your wish 😄